src/engraving/libmscore/cmd.cpp

```diff
@@ -26,7 +26,8 @@
 */
 
 #include <assert.h>
-
+#include <algorithm>
+#include <iostream>
 #include "translation.h"
```

Comment:
It looks like `<iostream>` is not necessary. Also, from a code-stylistic point of view, it is preferable to keep a newline between #includes from the standard library and our own #includes.
```suggestion
#include <algorithm>

#include "translation.h"
```

Reply:
Oh, thanks!
I missed that when removing my debug prints.

Reply:
Pushed an amended commit.
Should I mark this as resolved, or let you do that?

Reply:
Regarding marking conversations as solved, feel free to do that when you're confident you've made the correct change. 

For debug logging, you could also use our `LOGD() << …` (or `LOGI` or `LOGW` or `LOGE`). Sometimes that's more convenient, because it also prints the name of the method that's being called and it can handle some custom types like `mu::String` etc.